### PR TITLE
[ADF-2337] Fix viewer dark theme issues

### PR DESCRIPTION
**Pull request by popovicsandras**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
See https://issues.alfresco.com/jira/browse/ADF-2337

**What is the new behaviour?**
Fixed

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/core/viewer/components/imgViewer.component.scss [hunk 1]
```diff
@@ -22,7 +24,7 @@
 
             .adf-toolbar .mat-toolbar {
                 max-height: 48px;
-                background-color: mat-color($primary, default-contrast, 1);
+                background-color: mat-color($background, card, 0.87);
```
Comment (DenisVuyka) [inline lib/core/viewer/components/imgViewer.component.scss]: Is that the color that corresponds to the design spec?

Comment (DenisVuyka) [reply lib/core/viewer/components/imgViewer.component.scss]: Why Viewer needs to user the styles of Card?

Comment (popovicsandras) [reply lib/core/viewer/components/imgViewer.component.scss]: Do we have design spec for the dark theme?

Comment (popovicsandras) [reply lib/core/viewer/components/imgViewer.component.scss]: It doesn't need to use it, but this is the default panel background color in the material theming:

```js
$mat-dark-theme-background: (
  app-bar:    map_get($mat-grey, 900),
  background: #303030,
  card:       map_get($mat-grey, 800),
  dialog:     map_get($mat-grey, 800),
);
```

We don't have anything general for "panels", at least I don't know anything like that, except the card.

**Pull request merged**